### DISCUSSION TITLE
Tiny issue: SVG Emoji Favicons

### DISCUSSION THREAD

**Discussion by GarveScott-Lodge** [plausible/analytics#2129] [Anything Else]
This is an insignificant issue, but if you ever have some spare time, your dashboard isn't loading SVG Emoji favicons.

Using the site https://www.blackislecoal.co.uk/ as an example, this has a favicon created by an emoji within an SVG.

It's embedded using the code
&lt;link rel="icon" href="data:image/svg+xml,&lt;svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22&gt;&lt;text y=%22.9em%22 font-size=%2290%22&gt;&#128293;&lt;/text&gt;&lt;/svg&gt;"&gt;

![screenshot-plausible io-2022 08 22-16_01_04](https://user-images.githubusercontent.com/36659889/185953828-f9574553-76e4-4b34-a289-80ae349c72bc.png)

**Comment by metmarkosaric** [comment]
Thanks for reaching out @GarveScott-Lodge! We get these icons directly from DuckDuckGo and do not have any control over them. We basically showcase the same icon that they show in their search results/API. I just checked and their search result isn't loading your favicon either unfortunately. The thing is I'm not sure if DDG has a way for you to report this so they can fix it.

**Reply by GarveScott-Lodge** [reply]
That's great to know - I'll see if I can get hold of a contact email for DDG - I'm sure they'd like to resolve this if they can.

cheers

Garve
